catalyst/contrib/nn/modules/pooling.py

```diff
@@ -1,5 +1,7 @@
 # flake8: noqa
 # @TODO: code formatting issue for 20.07 release
```

Comment:
🤔 looks like we need to remove it for the correct test case...

Reply:
Sorry I did not get you. Does it mean it cannot be merged ? 🥺🥺

Reply:
yeah, the first row blocks code style formating for this file
a bit of legacy :

Reply:
@Scitator should I remove those comments, so that you will be able to merge it ?

Reply:
could u please merge with the dev branch? I could not see the test running

Reply:
I am merging with Dev branch. I ran the tests locally they ran.

Reply:
u could check the github CI logs output for more info about the run

Reply:
@Scitator Sorry to bother you this much. Since this is my first ever pull request. I was not sure. I have removed the problematic docstrings. Still on doing that 1  test is failing here is the error.


Oh no! 💥 💔 💥
1 file would be reformatted, 313 files would be left unchanged.
Error: Process completed with exit code 1.would reformat catalyst/contrib/nn/modules/pooling.py

PS: catalyst-check-codestyle and catalyst-make-codestyle are not giving any error locally. 

So looking for some advice from you.

Reply:
u could check the logs https://github.com/catalyst-team/catalyst/pull/1090/checks?check_run_id=2029039506

Reply:
there are lines +/- for codestyle

Reply:
I used the codestyle version you mentioned. It reformatted the files and I pushed them. Now I think the codestyle formatted is buggy for some reason. I have no idea what should I do. I have ran it three times. Manually updated stuff. For some reason its just not working. @Scitator @Ditwoo your help would be appreciated.

Reply:
could you please do `git pull origin dev`? and merge with the `dev` branch?

Reply:
looks like git has failed to merge you in the right way :)

Reply:
honestly speaking, I suggest you do the following:
- create a new branch from the `dev` one
- add to the https://github.com/catalyst-team/catalyst/blob/dev/catalyst/contrib/nn/modules/pooling.py your contributio0n
- install `catalyst-codestyle==21.03rc` and  run `make-catalyst-codestyle`
- make a pull request to the `dev` branch

this will reduce the amount of git merge things we have to care about ;)

Reply:
@Scitator Thank you I will follow the advice. I am closing this pull request and opening one on the Dev branch